style.css

```diff
@@ -23,12 +23,61 @@ body {
   padding-right: 24px;
   display: flex;
   place-content: space-between;
+  position: relative;
 }
 
 .menu-wrapper {
   max-width: 100%;
 }
 
+.mobile-menu {
+  cursor: pointer;
+}
+
+.mobile-menu-exit {
+  position: fixed;
+  right: 0;
+  padding: 10px;
+}
```

Comment:
- [ ] Kindly give some spacing on the close button and change the color to white. It is too close to the edges and not easily visible. For a good user experience, it's good you align it just as you did for the hamburger menu by giving some margin-top and right and giving it a color of white just as it's in the Figma design will make it more visible. What do you think? 

![image](https://user-images.githubusercontent.com/50941074/161812546-ccf07594-5393-4032-ba49-7af1a3f5e3ea.png)